modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/exception/ModuleExceptionHandler.java

```diff
@@ -53,6 +55,13 @@ public ModuleExceptionHandler(ComponentModel componentModel, ExtensionModel exte
           .name(errorDefinition.getType())
           .build());
 
+      String errorTypeDefinitionName;
```

Comment:
instead of doing this, you should just overwrite the `toString()` implementation in the adapter